app/models/organization.rb

```diff
@@ -11,6 +11,6 @@ def self.related_instructions_organizations(occupation_standard)
   end
 
   def self.urban_institute
-    @@urban_institute ||= Organization.find_by(title: "Urban Institute")
+    Organization.find_by(title: "Urban Institute")
```

Comment:
@FerPerales the memoization kept making the specs fail. I couldn't figure it out, so took it out for now. We can revisit next week.